src/lib/builders/popover/create.ts

```diff
@@ -180,13 +174,8 @@ export function createPopover(args?: CreatePopoverProps) {
 		},
 	});
 
-	function toggleOpen(triggerEl?: HTMLElement) {
-		open.update((prev) => {
-			return !prev;
-		});
-		if (triggerEl) {
-			activeTrigger.set(triggerEl);
```

Comment:
removing this line, fixed the tests in https://github.com/melt-ui/melt-ui/pull/1080
for some reason, in a recent PR https://github.com/melt-ui/melt-ui/pull/1049, we added `tick()` to solve the issue mentioned in the PR. But for some reason, that started failing the tests in https://github.com/melt-ui/melt-ui/pull/1080 and by removing the highlighted line of code, the tests started passing.